### ISSUE TITLE
env tfvar files are not pushed to repo gcp-projects

### ISSUE THREAD

**Issue by daniel-cit** [GoogleCloudPlatform/terraform-example-foundation-app#34]
## Expected Behavior

After executing the steps in [Run cloudbuild dev/npd/prd envs](https://github.com/GoogleCloudPlatform/terraform-example-foundation-app/tree/main/foundation-extension/4-projects#run-cloudbuild-devnpdprd-envs), the files:

- `development.auto.tfvars` 
- `non-production.auto.tfvars`
- `production.auto.tfvars`

are pushed to the repo `gcp-projects`

## Actual Behavior

Files are not pushed to the repo

## Steps to Reproduce the Problem

1. follow the steps in [4-projects](https://github.com/GoogleCloudPlatform/terraform-example-foundation-app/tree/main/foundation-extension/4-projects#usage)

## Specifications

- Version: d92e89485d2b6ea1caf633b66b41d8a10380e64d
- Platform: Ubuntu 20.04 LTS

## Extra Info

the README is missing a step with
```
Commit changes with git add . and git commit -m 'Your message'
```
just before pushing the code

**Comment by rutalreja-deloitte** [comment]
@daniel-cit 
development.auto.tfvars
non-production.auto.tfvars
production.auto.tfvars
are in their respective folders with the example.tfvars and there are instruction to rename them

**Comment by rutalreja-deloitte** [comment]
Extra Info addressed in #36 instruction was in readme though not in the right order

**Comment by bharathkkb** [comment]
fixed via #36
